crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -2028,6 +2028,8 @@ mod YASPoolTests {
         if token_0_swapped_amount == 0
             && token_1_swapped_amount == 0 { //this avoids 0/0 , no tokens swapped = exec_price: 0
             0
+        } else if token_0_swapped_amount == 0 {
+            '-Infinity'.into()
```

Comment:
Can you add a comment describing how this works? Something similar to what you mentioned in Slack.
Describing the different cases and the output for each one